common/pom.xml

```diff
@@ -17,7 +17,7 @@
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
             <version>1.18</version>
-            <scope>provided</scope>
+            <!--<scope>provided</scope> Velocity -->
```

Comment:
Will not relocating snakeyaml cause problems?

Reply:
I suspect it probably would if other plugins use different versions, it's not on Bungee / Bukkit as they provide a version that works.